docs/docker_deployment_guide.md

```diff
@@ -64,7 +71,10 @@ sudo chown $USER:$USER storage/media storage/caddy_config storage/caddy_data
 ### Configure `.env` and `compose.override.yml`
 
 1. Choose your Redis password, PostgreSQL password, RabbitMQ password, and Mercure password.
-2. Place them in the corresponding variables in both `.env` and `compose.override.yml`.
+2. Place the passwords in the corresponding variables in both `.env` and `compose.override.yml`.
+3. Update the `SERVER_NAME`, `KBIN_DOMAIN` and `KBIN_STORAGE_URL` in `.env`.
+4. Update `APP_SECRET` in `.env`, generate a new one via: `node -e  "console.log(require('crypto').randomBytes(16).toString('hex'))"`
```

Comment:
Nobody explain that you also need to set `APP_SECRET` and change those other variables in your `.env` file.